packages/quick-edit/functions/_middleware.ts

```diff
@@ -14,38 +14,41 @@ export const onRequest = async ({
 						: "Solarflare Light",
 				"workbench.startupEditor": "none",
 				"editor.minimap.autohide": true,
-			},
-			productConfiguration: {
-				nameShort: "Quick Edit",
-				nameLong: "Cloudflare Workers Quick Edit",
-				applicationName: "workers-quick-edit",
-				dataFolderName: ".quick-edit",
-				version: "1.76.0",
-				extensionsGallery: {
-					serviceUrl: "https://open-vsx.org/vscode/gallery",
-					itemUrl: "https://open-vsx.org/vscode/item",
-					resourceUrlTemplate:
-						"https://openvsxorg.blob.core.windows.net/resources/{publisher}/{name}/{version}/{path}",
-				},
-				extensionEnabledApiProposals: {
-					"cloudflare.quick-edit-extension": [
-						"fileSearchProvider",
-						"textSearchProvider",
-						"ipc",
-					],
-				},
-			},
-			additionalBuiltinExtensions: [
-				{
-					scheme: url.protocol === "https:" ? "https" : "http",
-					path: "/quick-edit-extension",
+				"files.exclude": {
+					"workers-types.d.ts": true,
```

Comment:
Could you change this to _all_ .d.ts files? I want to try and experiment with injecting an `worker-configuration.d.ts` with the type of your `Env` object at some point, which should also be hidden

Reply:
Sure, we should consider that we need to make it more granular after we support TS because people will want to make their own DTS files.

Reply:
Done 😄